src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,24 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        """
+        Populate package manifest with metadata as per EIP
+        https://eips.ethereum.org/EIPS/eip-2678#the-package-meta-object
+        """
+        meta = self.config_manager.get_config("meta")
+        try:
+            return PackageMeta(**meta.serialize())
```

Comment:
Try this:
```suggestion
        try:
            return self.config_manager.meta
```

Reply:
I was able to get the meta into the project by using this 
https://github.com/ApeWorX/ape/pull/802/commits/0b68bf1c4a476f49588eb7f0dc7be5d52e24dbcb

Reply:
still need to get the meta into the manifest